ppd/ppd-test.c

```diff
@@ -156,6 +156,10 @@ int ppdTest(int ignore,          // Which errors to ignore
   ppd = NULL;
   status = ERROR_NONE;
   root = rootdir;
+  if (root == NULL)
+  {
```

Comment:
Please do not use braces if there is only a single statement. It makes the code unnecessarily long ...